src/atoms/Text.tsx

```diff
@@ -13,6 +13,8 @@ const Text = styled.p<TextProps>`
     css`
       color: ${({ theme }) => theme.colors.blackLight};
       font-size: 16px;
+      font-weight: 600;
```

Comment:
This makes all paragraphs on the page bold (`font-weight: 600`). Are you sure that's intended?

Reply:
it is not for all paragraphs, just with props 'big'
 ${props =>
    props.big &&
    css`
      color: ${({ theme }) => theme.colors.blackLight};
      font-size: 16px;
      font-weight: 600;
      margin: 0;
    `}